tests/requirements.txt

```diff
@@ -3,6 +3,7 @@
 
 # Additional packages needed to run the tests.
 chromedriver-binary==2.45.0
+dash==0.40.0
```

Comment:
This will create a double (at the moment, redundant) requirement, because of `-r ../requirements.txt` (top of the file). I suggest replacing the latter with all requirements except for `dash`.

Reply:
Alternatively, revert d3993ea5f8a6683 because it's already ensured through (top-level) `requirements.txt`). This way we have a clear and healthy use of `setup.py` vs `requirements.txt` files.

Reply:
Done in 2dd952e -- can't wait to unblock the situation...